### PR TITLE
Update tuya.js

### PR DESCRIPTION
**Pull request by kloknibor**
Probably fixing:

#3076 #3082

### CODE REVIEW

**Comment by Koenkk** [comment]
This should only be done for the which do not support the configure reporting (since polling increases stress on the network and will provide less frequent updates). What are the `manufacturerName` of the devices that need this?

**Comment by Z18Aplha** [comment]
I agree to Koenkk. 
This device supports reporting.

**Comment by dmaasland** [comment]
The device reports itself as a TS011F with manufacturer _TZ3000_dpo1ysak. It does seem to behave differently for different people though. Perhaps a firmware difference or whitelabling issue. Mine does support power metering when queried manually via the dev console but haven't been able to get automatic reporting to work yet.

**Comment by kloknibor** [comment]
I changed it to only poll for the '_TZ3000_dpo1ysak' device. @Koenkk can you re-check this PR? Thanks!

File: devices/tuya.js [hunk 1]
```diff
@@ -788,6 +788,43 @@ module.exports = [
...
+        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3000_dpo1ysak'},
+        model: 'TS011F_plug',
+        description: 'Smart plug (with power monitoring by polling)',
+        vendor: 'TuYa',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report, fz.tuya_switch_power_outage_memory],
+        toZigbee: [tz.on_off, tz.tuya_switch_power_outage_memory],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'haElectricalMeasurement', 'seMetering']);
+            await reporting.rmsVoltage(endpoint, {change: 5});
```
Comment (Koenkk) [inline devices/tuya.js:801]: I think all reporting setup can be removed here (it doesn't work anyway)

Comment (kloknibor) [reply devices/tuya.js]: like this?

**Comment by kloknibor** [comment]
Like this?

**Comment by JohnRabbit1** [comment]
I think line 769 should be deleted (but probably you already did)

**Comment by Koenkk** [comment]
Thanks!

**Comment by KMDobrev** [comment]
Hallo,

I just did an update.  But still no energy measurement.

Screenshot_20211101-231748
Screenshot_20211101-231736

**Pull request merged**